pkg/controller/jivavolume/jivavolume_controller.go

```diff
@@ -59,12 +60,17 @@ var (
 )
 
 const (
-	pdbAPIVersion = "policyv1beta1"
+	pdbAPIVersion            = "policyv1beta1"
```

Comment:
Why is this variable called pdb?

Reply:
This represents the api version for pod disruption budget.